content/articles/understanding-invoice.markdown

```diff
@@ -0,0 +1,73 @@
+
+|    | **Name**           | **Description**                                                                                                                                                                       |
+|:--:|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 1. | **DNSimple**       | We are focused on providing [effortless domain and DNS management](https://support.dnsimple.com/articles/dnsimple-services/).                                                         |
+| 2. | **Invoice Number** | The unique number assigned to your invoice.                                                                                                                                           |
+| 3. | **Invoice Date**   | The date the invoice was created. The invoice date is the day of the start of your billing cycle. For example, if your billing cycle is Jan 15 - Feb 14, your Invoice Date is Jan 15. |
+| 4. | **Terms**          | Indicates whether your invoice is `Due Upon Receipt`, `Open`, or `Paid`.                                                                                                              |
+| 5. | **To**             | Your company name and address. To update this information, see [Billing Settings](/articles/billing-settings/).                                                                       |
+| 6. | **Your Order**     | This section gives an itemized listing of your [charges](#charge-types) for the invoice billing cycle cycle.                                                                  |
+| 7. | **Subtotal**       | The total amount for the invoice before any credits or discounts.                                                                                                                     |
+| 8. | **Credit**         | The amount of any credit deducted from your [account credit balance](/articles/account-subscription-balance/).                                                                        |
+| 9. | **Total**          | The amount charged to your credit card for the invoice billing cycle.                                                                                                                                         |
+
+
+## Charge types
+
+DNSimple has both recurring charges and one-time charges. One-time charges, such as the fees you pay for registering a domain or for purchasing a Standard or Wildcard SSL certificate, appear on a separate invoice than the fees for your subscription. One-time charges _can_ be recurring, such as if you've enabled auto-renewal for a domain.
+
+### Recurring charges
+
+- DNSimple subscription
+- [Email forwards](/articles/email-forwarding/)
+- [Extra domain zones](/articles/dns-hosting/#extra-zones)
+
+![Recurring charges](/files/recurring-charges.png)
+
+### One-time charges
```

Comment:
We frequently refer to these 1 time charges as "purchases". Just bringing this up in case we want to also have this term in this article.